.gitmodules

```diff
@@ -3,11 +3,18 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/LayerZero-v2"]
 	path = lib/LayerZero-v2
-	url = https://github.com/JorgeAtPaladin/LayerZero-v2
-	branch = lz-upgrade
+	url = https://github.com/dan13ram/layerzero-v2-upgradeable
```

Comment:
Uses my fork of this repo

https://github.com/dan13ram/layerzero-v2-upgradeable

Which is one commit ahead, diff below:
https://github.com/JorgeAtPaladin/LayerZero-v2/compare/lz-upgrade...dan13ram:layerzero-v2-upgradeable:main